### DISCUSSION TITLE
cannot change the repository name

### DISCUSSION THREAD

**Discussion by ypwang0371** [alshedivat/al-folio#374] [Q&A] [answered]
Hello, I am new to GitHub and forked the theme to my own account to create my personal website. I found that GitHub does not allow me to change my repository name. I was wondering if anyone could help me. Thank you so much!
![image](https://user-images.githubusercontent.com/89311305/130329069-5faabe9c-37e4-4186-ac34-2337ae5c4aa4.png)

**Comment by rohandebsarkar** [comment, marked as answer]
Another repository with the same name exists on your GitHub account. Repository names must be unique. Therefore, delete the other repository, then try again.

**Reply by ypwang0371** [reply]
Thank you!!
